### PR TITLE
LPS-75377 As a System Administrator I can browse the available system settings following a semantic organization that facilitates finding configurations by browsing

### PR DESCRIPTION
**Pull request by pei-jung**
Hey Jorge,

Can you check if my changes are ok? If everything looks good, we can send it to Brian. Thanks

### CODE REVIEW

**Comment by pei-jung** [comment]
ci:test

**Comment by JorgeFerrer** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/display/ConfigurationCategoryMenuDisplay.java [hunk 1]
```diff
@@ -71,15 +75,8 @@ private void _addConfigurationModel(ConfigurationModel configurationModel) {
 		configurationScopeDisplay.add(configurationModel);
 	}
 
-	private static final String[] _SCOPE_KEYS = {
```
Comment (JorgeFerrer) [inline modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/display/ConfigurationCategoryMenuDisplay.java]: The reason for having this constant was to specify the desired order in the UI. Although it probably wasn't obvious. I'm going to think of a better name for the constant  to make it more clear and will add it back. 
I will leave the rest of your changes.

**Comment by JorgeFerrer** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/55120. [See changes here.](https://github.com/JorgeFerrer/liferay-portal/compare/f96bcb8e37c4~1...f96bcb8e37c4) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by JorgeFerrer** [comment]
Thanks for your review @pei-jung, I just made one change and submitted it to Brian.

**Pull request closed without merging**